sdcm/nemesis.py

```diff
@@ -776,9 +776,18 @@ def _destroy_data_and_restart_scylla(self):
         self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
 
         try:
+            tables = []
+            for table in ks_cfs:
+                has_data = self.cluster.is_table_has_data(db_node=self.target_node, table_name=table)
+                if has_data:
+                    tables.append(table)
+                if len(tables) > 20:
```

Comment:
you could in this case stop when `len(tables) > 5` as only 5 will be used, no?

Reply:
since it's choosing a random one each time, it cut it a bit more slack, and have a bigger selection to random from.

Anyhow the whole idea is that is would do that 5000 times...